signbank/dictionary/csv_import.py

```diff
@@ -695,8 +695,9 @@ def confirm_import_qualtrics_csv(request):
 
             ))
 
-        ValidationRecord.objects.bulk_create(validation_records_added)
-        TaggedItem.objects.bulk_create(bulk_tagged_items)
+        # ignoring conflicts so the unique together on the model filters out potential duplicates
+        ValidationRecord.objects.bulk_create(validation_records_added, ignore_conflicts=True)
```

Comment:
**note** This feels a bit cheeky (asking for forgiveness, not permission), but it works so 🤷🏻‍♀️